README.md

```diff
@@ -7,10 +7,13 @@ Requirements:
 
 ####To install:
 
-* git clone https://github.com/poldracklab/pydeface.git
+* git clone https://github.com/jmtyszka/pydeface.git
 * cd pydeface
```

Comment:
Agreed - this is legacy from my branch. Sorry I missed it before making the pull request.